DSCResources/VE_XD7Administrator/VE_XD7Administrator.psm1

```diff
@@ -59,14 +59,15 @@ function Get-TargetResource {
 
         if ($Credential) {
             AddInvokeScriptBlockCredentials -Hashtable $invokeCommandParams -Credential $Credential;
+            Write-Verbose ($localizedData.InvokingScriptBlockWithParams -f [System.String]::Join("','", @($Name, $Enabled, $Ensure)));
```

Comment:
Can you move the Write-Verbose call above the `if ($Credential) {` call? This will ensure the verbose logs is output whether `-Credential` or `-PSDscRunAsCredential` is used.